jekyll/_layouts/classic-docs.html

```diff
@@ -46,7 +46,12 @@
                   {% assign nowTimestamp = 'now' | date: '%s' %}  
                   {% assign timestampDiff = nowTimestamp | minus: dateStart %}
                   {% assign yearInSeconds = 60 | times: 60 | times: 24 | times: 365 %}
-                  {% if timestampDiff > yearInSeconds %} 1+ year ago {% else %} {{ page.last_modified_at | timeago }} {% endif %}
+                  {% assign ghLink = 'https://github.com/circleci/circleci-docs/commits/master/jekyll/' | append: page.path %}
```

Comment:
https://github.com/circleci/circleci-docs/blob/ac9317d8009b13191b348c01959a2978da65be19/jekyll/_config.yml#L10

Reply:
Oh nice! Thanks!

Reply:
This code has been moved to quick-info-list.html see [link](https://github.com/circleci/circleci-docs/pull/6077)